internal/asm/f64/stubs_amd64.go

```diff
@@ -170,3 +170,29 @@ func ScalIncTo(dst []float64, incDst uintptr, alpha float64, x []float64, n, inc
 //      sum += x[i]
 //  }
 func Sum(x []float64) float64
+
+// L2NormUnitary is
```

Comment:
Have this first line be `// L2NormUnitary is the level 2 norm of x:`. This is the first function we have here where the Go is non-trivial, so I think it needs to be spelled out. Can you also reflect this in the doc comment for the pure-Go version.